cns/restserver/ipam.go

```diff
@@ -290,35 +255,43 @@ func (service *HTTPRestService) updateEndpointState(ipconfigsRequest cns.IPConfi
 }
 
 // releaseIPConfigHandlerHelper validates the request and removes the endpoint associated with the pod
-func (service *HTTPRestService) releaseIPConfigHandlerHelper(ctx context.Context, ipconfigsRequest cns.IPConfigsRequest) (*cns.Response, error) {
+func (service *HTTPRestService) releaseIPConfigHandlerHelper(ctx context.Context, ipconfigsRequest cns.IPConfigsRequest) (*cns.IPConfigsResponse, error) {
```

Comment:
This is so that both the release and request handler functions for ipconfigsrequest would have the same function signature that is `IPConfigsHandlerFunc`, which make it easier when implementing the middleware. Also there is not much of a different between how `cns.Response` is structured compared to `cns.IPConfigsResponse`.